### ISSUE TITLE
how to set the m and s when there are only 1000 identities?

### ISSUE THREAD

**Issue by Chanbluky** [happynear/AMSoftmax#27]
thanks very much for your contribution. i am using gray images to train the model with only about 1000 identities, how need i set the m and s. both of them should be set smaller?

**Comment by happynear** [comment]
It's difficult to give you a number. You may try [AM Softmax without feature normalization](https://github.com/happynear/AMSoftmax/blob/master/prototxt/face_train_test_wo_fn.prototxt) or [Auto AM Softmax](https://github.com/happynear/AMSoftmax/tree/master/prototxt/auto) to avoid setting some of the hyper-parameters.

**Comment by happynear** [comment]
But the best accuracy still should be gotten by carefully manually tuned hyper-parameters.

**Comment by Chanbluky** [comment]
thanks for your quickly reply! by the way, during the training on VGGFACE2 dataset, how did you define the validation set? select some image from the training set?  what was the final loss and the accuracy on the validation set? thanks!

**Comment by happynear** [comment]
Don't care about the loss and accuracy. They are meaningless.
Face verification usually use verification rate or TPR@low FAR as the criteria, so validate on LFW or other small testing datasets such as BLUFR, AgeDB, CFP etc.

**Comment by Chanbluky** [comment]
thanks a lot!
